src/Service/Request/InitializeRequestObjectCreator.php

```diff
@@ -68,6 +69,7 @@ public function create(
         $totalPrice = (int) (round($totalPrice));
         $payment = $this->requestObjectCreator->createPayment($cart, $totalPrice);
         $payer = new Payer();
+        $payer->setLanguageCode($cart->getAssociatedLanguage()->iso_code);
```

Comment:
so this $payer is not associated with any of the returned result? How this is being transmitted to the API?